specs/Standards/Sould Bound Token/README.md

```diff
@@ -0,0 +1,456 @@
+---
+NEP: 393
+Title: Soulbound Token
+Authors: Robert Zaremba <@robert-zaremba>
+DiscussionsTo:
+Status: Draft
+Type: Standards Track
+Category: Contract
+Created: 12-Sep-2022
+Requires:
+---
+
+# NEP: Soulbound Token
+
+## Summary
+
+Soulbound Token (SBT) is a form of a NFT which represents an aspect of an account: _soul_. Transferability is limited only to a case of recover-ability or transferring the soul - a _soul transfer_ should coordinate with a registry to transfer all SBTs from one account to another, and _killing_ the source account.
+
+SBTs are well suited for carrying proof-of-attendance, proof-of-unique-human "stamps" and other similar credibility-carriers.
+
+## Motivation
+
+Recent [Decentralized Society](https://www.bankless.com/decentralized-society-desoc-explained) trends open a new area of Web3 research to model various aspects of what characterizes humans. Economic and governance value is generated by humans and their relationship. SBTs can represent the commitments, credentials, and affiliations of “Souls” that encode the trust networks of the real economy to establish provenance and reputation.
+
+> More importantly, SBTs enable other applications of increasing ambition, such as community wallet recovery, Sybil-resistant governance, mechanisms for decentralization, and novel markets with decomposable, shared rights. We call this richer, pluralistic ecosystem “Decentralized Society” (DeSoc)—a co-determined sociality, where Souls and communities come together bottom-up, as emergent properties of each other to co-create plural network goods and intelligences, at a range of scales.
+
+Creating a strong primitives is necessary to model new innovative systems and decentralized societies. Examples include reputation protocols, undercollateralized lending, one-person-one-vote, fair airdrops & ICOs, universal basic income, non KYC identity systems, Human DAOs and methods for Sybil attack resistance.
+
+We propose a SBT standard to model protocols described above.
```

Comment:
BTW, "a SBT" -> "an SBT" though the whole text. I suggest to put the text into Grammarly to fix other spelling as well.